### PR TITLE
e2e: enable using Qemu monitor

### PR DESCRIPTION
**Pull request by askervin**

### CODE REVIEW

File: demo/lib/host.bash [hunk 1]
```diff
@@ -160,6 +161,7 @@ host-wait-vm-ssh-server() {
         sleep 5
         VM_IP=$(${GOVM} ls | awk "/$VM_NAME/{print \$4}")
     done
+    echo "# VM SSH server  : ssh $VM_SSH_USER@$VM_IP"
```
Comment (askervin) [inline demo/lib/host.bash:164]: This questionable whitespace aligns the colon with the above lines.

File: demo/lib/host.bash [hunk 2]
```diff
@@ -150,6 +150,7 @@ host-create-vm() {
     else
         echo "Warning: cannot verify Qemu version on govm image. In case of failure, check it is >= 5.0" >&2
     fi
+    echo "# VM Qemu monitor: docker exec -it $VM_CONTAINER_ID nc local:/data/monitor"
```
Comment (askervin) [inline demo/lib/host.bash:153]: This is a way to connect to Qemu monitor without sudo in order to access `-monitor unix:socket-file` or change network namespace to connect to `-monitor tcp::port`.

**Review by klihub — APPROVED** [review]
LGTM

**Review by kad — APPROVED** [review]
lgtm

**Pull request merged**